src/meta/expr.lean

```diff
@@ -725,6 +710,12 @@ meta def drop_pis : list expr → expr → tactic expr
 | [] e := return e
 | _  _ := failed
 
+/-- `instantiate_pis es e` instantiates the pis in `e` with the expressions from `es`.
+  Does not check whether the result remains type-correct. -/
+meta def instantiate_pis : list expr → expr → expr
+| (list.cons v vs) (pi n bi d b) := instantiate_pis vs (b.instantiate_var v)
```

Comment:
Why not just
```suggestion
| (v :: vs) (pi n bi d b) := instantiate_pis vs (b.instantiate_var v)
```

Reply:
I copied it from another definition. Fixed